src/rust/engine/process_execution/bazel_protos/build.rs

```diff
@@ -1,29 +1,99 @@
-use std::env::home_dir;
```

Comment:
Ditto the `unwrap` vs `expect` comment above

Reply:
The only remaining `unwrap`s after simplification are can't-happens afaict. IE known Pants repo path names being converted to UTF-8, listing of directories known to exist in the repo, etc. Let me know if we have different ideas about this though.